CMakeLists.txt

```diff
@@ -256,6 +256,13 @@ if(BUILD_TESTS)
       ADDITIONAL_ARGS
         --app-script ${CMAKE_SOURCE_DIR}/src/apps/logging/logging.lua)
 
+    ## Lua sample app end to end test
```

Comment:
I'm not sure if running this for every CI run is what we want. We definitely want it to run often enough to not have it rot though. Perhaps only run it first thing in the morning? Thoughts?

Reply:
I think it's a good test case for the lua side of CCF as it is bigger than the logging lua client test and does quite a bit more than that one. Not sure how it would be restricted to run only in the morning though without breaking it out to another ci pipeline?

Reply:
How long does it take to run?

Reply:
5-6 seconds

Reply:
Let’s run it every time then. When we have too many, we’ll start running some concurrently.